tests/contenttypes_tests/tests.py

```diff
@@ -21,7 +21,7 @@
 from django.utils.encoding import force_str, force_text
 
 from .models import (
-    Article, Author, ModelWithNullFKToSite, Post, SchemeIncludedURL,
+    Answer, Article, Author, ModelWithNullFKToSite, Post, Question, SchemeIncludedURL,
```

Comment:
However all tests pass even though the line length is getting corrected when I am running isort. I'm not sure why that is. 
Also I am getting a lot of changes like this after running isort so I can't really use it. I am not sure why that happens. Do you have an idea? Do I need to use an older version of isort perhaps or something else?
```
-from .models import A, B, Bar, D, DataPoint, Foo, RelatedPoint
+from .models import A, B, D, Bar, DataPoint, Foo, RelatedPoint
```